src/shared/behaviors/nonscrollable.html

```diff
@@ -7,6 +7,35 @@
 
 */
 (function(scope) {
+
+	var _openedInstances = [];
+	var __oldBodyPointer;
+
+	function _addOpenedInstance(instance) {
+		_openedInstances.push(instance);
+	}
+
+	function _removeOpenedInstance(instance) {
+		_openedInstances.splice(_openedInstances.indexOf(instance), 1);
```

Comment:
I'd recommend checking that `.indexOf()` didn't return -1 (i.e. instance-not-found)